must-gather/collection-scripts/gather_visualization_resources

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
same comment as above

Reply:
Resolving this comment in favour of eo-thread above.